components/builder-web/app/BuilderApiClient.ts

```diff
@@ -234,9 +234,18 @@ export class BuilderApiClient {
             }).then(response => {
                 if (response.ok) {
                     resolve(true);
-                // Getting a 404 means the user does not exist.
                 } else if (response.status === 404) {
-                    reject(new Error(`User '${username}' does not exist`));
+                    let err = new Error(`User '${username}' does not exist`);
+
+                    response.text().then(msg => {
+                        if (msg.match(/valid GitHub user/)) {
```

Comment:
This is an example of why the server should return something more machine readable and not something for the user. We *really* don't ever want to gain additional context from an error by using a regex - not just because of localization but because it's really easy to break the client if we change the format of the message on the server.